relay-server/src/actors/outcome_aggregator.rs

```diff
@@ -139,19 +139,22 @@ impl OutcomeAggregator {
                 category,
             } = bucket_key;
 
-            let timestamp = UnixTimestamp::from_secs(offset * bucket_interval);
-            let outcome = TrackOutcome {
-                timestamp: timestamp.as_datetime(),
-                scoping,
-                outcome,
-                event_id: None,
-                remote_addr,
-                category,
-                quantity,
-            };
-
-            relay_log::trace!("Flushing outcome for timestamp {}", timestamp);
-            outcome_producer.send(outcome);
+            if let Some(timestamp) =
+                UnixTimestamp::from_secs(offset * bucket_interval).as_datetime()
```

Comment:
_nit_: Should we fall back to some timestamp, like `::now()`? Otherwise, we're dropping an outcome which sounds worse than reporting it at the wrong time. 

Applies to all changes in this PR.